doc/api/crypto.markdown

```diff
@@ -627,17 +627,26 @@ You can get a list of supported digest functions with
 
 Synchronous PBKDF2 function.  Returns derivedKey or throws error.
 
-## crypto.randomBytes(size[, callback])
+## crypto.randomBytes(size, callback)
 
 Generates cryptographically strong pseudo-random data. Usage:
 
-    // async
     crypto.randomBytes(256, function(ex, buf) {
       if (ex) throw ex;
       console.log('Have %d bytes of random data: %s', buf.length, buf);
     });
 
-    // sync
+NOTE: This will block if there is insufficient entropy, although it should
+normally never take longer than a few milliseconds. The only time when this
+may conceivably block is right after boot, when the whole system is still
+low on entropy.
+
+NOTE: The synchronous API for this method is deprecated - please use `crypto.randomBytesSync(size)` instead.
+
+## crypto.randomBytesSync(size)
+
+A synchronous version of `crypto.randomBytes(size)`. Usage:
```

Comment:
It kinda gives me a feeling that there are more than one synchronous implementations and this is one of them.